dev/Deployment/DeploymentTraceLogging.h

```diff
@@ -67,14 +67,14 @@ class WindowsAppRuntimeDeployment_TraceLogger final : public wil::TraceLoggingPr
                 TraceLoggingValue(deploymentErrorExtendedHResult, "DeploymentErrorExtendedHResult"),
                 TraceLoggingValue(deploymentErrorText, "DeploymentErrorText"),
                 TraceLoggingValue(deploymentErrorActivityId, "DeploymentErrorActivityId"),
-                TraceLoggingValue(isFullTrustMicrosoftPublisherPackage, "isFullTrustMicrosoftPublisherPackage"));
+                TraceLoggingValue(isFullTrustPackage, "isFullTrustPackage"));
```

Comment:
Would it be more useful to log `isPackagedProcess` and `integrityLevel` instead of the compound computed answer?

Reply:
Yeah.. was just thinking the same. Sure, I will log those two instead of the compound.